src/components/Calls/CallNotification.js

```diff
@@ -34,31 +38,38 @@ export const CallNotification = ({ callProvider, callerUsername, url, dispatch }
 		await dispatch({ incomingCallAlert: null });
 	};
 
+	const timeout = () => {
+		setTimeout(async function() { setShow(!{ show }); }, 20000);
+	};
+
 	return (
-		<div className={createClassName(styles, 'call-notification')}>
+		<div>
 			{ show
 				? (
-					<div className={createClassName(styles, 'call-notification__content')}>
-						<div className={createClassName(styles, 'call-notification__content-avatar')}>
-							<Avatar
-								src={getAvatarUrl(callerUsername)}
-								large
-							/>
-						</div>
-						<div className={createClassName(styles, 'call-notification__content-message')}>
-							{ I18n.t('Incoming video Call') }
-						</div>
-						<div className={createClassName(styles, 'call-notification__content-actions')}>
-							<Button onClick={declineClick} className={createClassName(styles, 'call-notification__content-actions-accept')}>
-								<PhoneDecline width={20} height={20} /> <span style='margin-left:5px'> {I18n.t('Decline')} </span>
-							</Button>
-							<Button onClick={acceptClick} className={createClassName(styles, 'call-notification__content-actions-decline')} >
-								<PhoneAccept width={20} height={20} /><span style='margin-left:5px'> {I18n.t('Accept')} </span>
-							</Button>
+					<div className={createClassName(styles, 'call-notification')}>
+						<div className={createClassName(styles, 'call-notification__content')}>
+							<div className={createClassName(styles, 'call-notification__content-avatar')}>
+								<Avatar
+									src={getAvatarUrl(callerUsername)}
+									large
+								/>
+							</div>
+							<div className={createClassName(styles, 'call-notification__content-message')}>
+								{ I18n.t('Incoming video Call') }
+							</div>
+							<div className={createClassName(styles, 'call-notification__content-actions')}>
+								<Button onClick={declineClick} className={createClassName(styles, 'call-notification__content-actions-decline')}>
+									<PhoneDecline width={20} height={20} /> <span style='margin-left:5px'> {I18n.t('Decline')} </span>
+								</Button>
+								<Button onClick={acceptClick} className={createClassName(styles, 'call-notification__content-actions-accept')} >
+									<PhoneAccept width={20} height={20} /><span style='margin-left:5px'> {I18n.t('Accept')} </span>
+								</Button>
+							</div>
 						</div>
 					</div>
 				)
 				: null
 			}
+		    {timeout()}
```

Comment:
Instead of calling this function directly, why not use some react events like `componentDidMount()` or something else and call `setTimeout` there